core/typedefs.h

```diff
@@ -135,17 +129,16 @@ inline void __swap_tmpl(T &x, T &y) {
 	y = aux;
 }
 
-#endif //swap
+#endif // Swap
 
-/* clang-format off */
+// clang-format off
 #define HEX2CHR(m_hex) \
 	((m_hex >= '0' && m_hex <= '9') ? (m_hex - '0') : \
 	((m_hex >= 'A' && m_hex <= 'F') ? (10 + m_hex - 'A') : \
 	((m_hex >= 'a' && m_hex <= 'f') ? (10 + m_hex - 'a') : 0)))
-/* clang-format on */
+// clang-format on
```

Comment:
Perhaps we should keep it as `/* clang-format off */` and `/* clang-format on */`, for consistency with the how the rest of the engine's files use it?